infra/app/api.bicep

```diff
@@ -20,7 +20,6 @@ module api '../core_local/host/appservice.bicep' = {
     name: name
     location: location
     tags: union(tags, { 'azd-service-name': serviceName })
-    alwaysOn: false
```

Comment:
intentionally dropped?

Reply:
This is duplicate (seems some merge issue put it in hardcoded as well as as parameter